docs/how_to_knit/basics.md

```diff
@@ -1,22 +1,35 @@
 
 ## Single Bed Fair Isle
-For your first practice piece, load a test pattern such as:
+
+To create your own patterns, refer to the AYAB manual, section "pattern_image_creation". Make sure the 
+width of the pattern has an even number of pixels (=stitches).  
+
+For your first practice sample, use this Triangles image that also comes with
+AYAB 0.9, find it in Documents>AYAB>patterns
 
 ![triangles_60x10.png](img/dbj_2-color/triangles_60x10.png)
 
+To create your own patterns, refer to the AYAB manual, section "pattern_image_creation". 
+Use thin yarns, you'll be using side-by-side needles on both beds. (for those
+in the US, Woollike yarn works well.). 
+ 
+
 1. Cast on using needles Left30 - Right30 and knit a few rows of stockinette, ending on the left 
    ([KH910 manual](http://machineknittingetc.com/brother-kh910-user-guide.html), pp. 10-15, 18-21).
 2. Position the carriage so that the magnet on the center back of the carriage is outside the left turn mark.
-3. Leave all the settings as they are. Single Bed, Color 2, Start Row 1, Start Needle 30, Stop Needle 30, Alignment Center.   
-4. Click Configure, wait a moment.
-5. Click Knit. Wait until it says “Please init machine…”
-6. Set the knob to KC1 or KC2.
-7. Move the carriage so the magnet on the back of the carriage crosses the left turn mark, and STOP.
-8. **IMPORTANT!!!** Wait 2 or 3 seconds, there will be a “triple-beep”
-9. Continue across the row. After you pass the end of the selected needles, it should beep.
+3. Run the AYAB software. Open a black/white raster image (preferrably in .PNG file format) in AYAB. 
```

Comment:
In line 7, usage of triangles_60x10.png is proposed, so I guess we should tell the user to open this specific file here?